.github/CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+# Python
```

Comment:
Is this 100% staff or is there a way to delegate to trusted community members?

Reply:
It's 100% staff. This prevents malicious changes to the GitHub Actions that could disclose a secret, like the packaging org auth URL.